sapl/materia/views.py

```diff
@@ -322,6 +322,26 @@ def get(self, request, *args, **kwargs):
             return self.get_redirect_deactivated()
 
 
+@permission_required('materia.detail_materialegislativa')
+def recuperar_proposicao(request):
+    data = request.GET['data']
```

Comment:
Se `data` não existir pode dar exceção, trocar por:

```suggestion
    data = request.GET.get['data']
```